src/deps.deno.ts

```diff
@@ -9,23 +9,20 @@ export type { File } from "https://lib.deno.dev/x/grammy@v1/types.ts";
 // Determine whether a file path is absolute
 export { isAbsolute as isAbsolutePath } from "https://deno.land/std@0.135.0/path/mod.ts";
 // Perform a request and returns the raw body
-async function fetchResponseBody(url: string) {
+export async function fetchFile(url: string) {
     const { body } = await fetch(url);
     if (body === null) throw new Error("Download failed, no response body!");
     return body;
 }
 // Create a temporary file, and copy a local file to a file path
 export const { makeTempFile: createTempFile, copyFile } = Deno;
-// Streams the repsonse body of a URL
-export async function* streamFile(url: string) {
-    const body = await fetchResponseBody(url);
-    for await (const chunk of body) {
-        yield chunk;
-    }
+export async function* readFile(path: string): AsyncIterable<Uint8Array> {
+    const file = await Deno.open(path);
+    yield* file.readable;
```

Comment:
```suggestion
export async function readFile(path: string): AsyncIterable<Uint8Array> {
    const file = await Deno.open(path);
    return file.readable;
```

Reply:
Will do that in v2; this keeps the return type consistent among `deps.*.ts`.

Reply:
Now that I think about it, I don't think we have to release this for 1.x. We can change the target branch to a new `v2` branch and ship these changes alongside the necessary adjustment that we will perform for grammY 2.0.

Reply:
I have bigger refactoring plans :) Please release a new minor version, I don't think I'm permitted to.

Reply:
You still could've applied these changes. They don't affect the return type.

Reply:
They do! Return type of Deno's `readFile` would become `Promise<AsyncIterable<Uint8Array>>`.

Reply:
Meh